test/JDBC/input/BABEL-1466-vu-verify.sql

```diff
@@ -0,0 +1,57 @@
+-- CONVERT from int to smallint is immutable/deterministic and can be used to generate computed column
+alter table babel_1466_vu_prepare_t1 add b as convert(smallint, 1) persisted;
```

Comment:
why the alters in verify?

Reply:
These check for the volatile functions cannot be added. They act as a kind of negative test